src/python/test_LossFunctionTranslation.py

```diff
@@ -0,0 +1,127 @@
+import unittest
+import json
+from vehicle import LossFunctionTranslation
+from tensorflow import keras
+
+class TestLossFunctionTranslation(unittest.TestCase):
+    def load_json(self, file_name):
+        path_to_json = f'/Users/marcocasadio/Projects/vehicle/src/python/test_json/{file_name}.json'
```

Comment:
Hmm so you've got an absolute path here so this won't work on anyone else's computer. You need to make it a relative path!